src/screens/MemberDetail/MemberDetail.tsx

```
@@ -265,14 +313,37 @@
           setShow={setShow}
         />
       )}
+      <div
+        className={`bg-white gap-4 ${styles.allRound} ${styles.navigationBox}`}
+      >
+        {navItems.map(({ to, icon, label }) => (
+          <div className={`bg-white gap-4 ${styles.allRound}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)